demo/asr/run.py

```diff
@@ -6,14 +6,17 @@
 
 def transcribe(audio):
     sr, y = audio
+    # Convert to mono if stereo
+    if y.ndim > 1:
+        y = y.mean(axis=1)
     y = y.astype(np.float32)
     y /= np.max(np.abs(y))
 
     return transcriber({"sampling_rate": sr, "raw": y})["text"]  # type: ignore
 
 demo = gr.Interface(
     transcribe,
-    gr.Audio(sources=["microphone"]),
+    gr.Audio(sources="microphone", type="numpy"),
```

Comment:
`type="numpy"` is the default so I don't think we need it

Reply:
Thanks, removed that
I admit I did this pull request quite fast as soon as I fixed it, so I didn't check if this parameter was necessary